### ISSUE TITLE
Call $sope.$apply() after user initiated onpopupclose event

### ISSUE THREAD

**Issue by alex-pl** [MozillaDE/MozMap#5]
Call $sope.$apply() at the end of a user initiated (here: only when the user has clicked on the x inside the popup) onpopupclose event.

Currently the user selection in the sidebar does not get removed when a user closes the pop up this way. This is because the scope doesn't get updated in that case.

**Comment by MichaelKohler** [comment]
I didn't check the code, but $scope.$apply() is the last resort if two-way-binding doesn't work. just my 2 cents

**Comment by alex-pl** [comment]
Yes you're right, but in this case it's necessary because the pop up and it's event comes from the Mapbox/Leaflet code which is not connected to Angular.
It's only the single case when the user closes the pop up by clicking the small "x" button.

**Comment by MichaelKohler** [comment]
Go ahead :)
